src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -559,6 +565,22 @@ private void RenderUnboundAttribute(string attributeName, Chunk attributeValueCh
             }
         }
 
+        private void RenderPreallocatedBoundAttribute(string attributeName, Chunk attributeValueChunk)
```

Comment:
Assuming [this](https://github.com/aspnet/Razor/pull/765/files#r64953504) is true. Inline this method.